src/signal.rs

```diff
@@ -131,10 +131,22 @@ pub fn os_signals() -> impl Stream<Item = SignalTo> {
     async_stream::stream! {
         loop {
             let signal = tokio::select! {
-                _ = sigint.recv() => SignalTo::Shutdown,
-                _ = sigterm.recv() => SignalTo::Shutdown,
-                _ = sigquit.recv() => SignalTo::Quit,
-                _ = sighup.recv() => SignalTo::ReloadFromDisk,
+                _ = sigint.recv() => {
+                    info!(message = "SIGINT signal received.");
```

Comment:
```suggestion
                    info!(message = "Signal received.", signal = "SIGINT");
```

Our typical logging style puts the "dynamic" bits as fields. This makes it easier to group/aggregate the logs later if desired without string parsing.